src/utils/parsers.rs

```diff
@@ -60,6 +61,47 @@ pub fn parse_graph_ref(graph_id: &str) -> Result<GraphRef> {
     }
 }
 
+#[derive(Debug, Serialize, Default, Clone)]
+pub struct ValidationPeriod {
+    pub from: Option<String>,
```

Comment:
These two are definitely `Option<String>`? I would have expected a u64 of sort, since it represents seconds in time.

Reply:
Yeah 😞 Their actual type in the API is an optional `Timestamp` which they define as:

> ISO 8601, extended format with nanoseconds, Zulu (or '[+-]seconds' for times relative to now)

But for our cases, that means an optional string.

Reply:
Ah! Interesting. Sounds good then!